### DISCUSSION TITLE
Tem uma pessoa conhecida minha mais com nick diferente , vamos ver o que vai dar obrigado

### DISCUSSION THREAD

**Discussion by luis007007** [community/community#132328] [Other Feature Feedback, Questions, & Ideas] [answered]
Vou esquecer eu achei elas e como eskasbetavam pegando pesado como deletaram tudo.   ### Select Topic Area

Question

### Body

estou sendo acesssadfo por todos os lados, tenho 3 emails e todos estao ao poder de quem esta acrecivamnete me axcessando, eu sei quem é porem tem uma organização junta dando apoio, e posivel eu localizar esta pessoal e falar para ela pegar leve?

**Comment by LiteBrite82** [comment, marked as answer]
Hi @luis007007 , thanks for participating!

Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
